src/services/alert/MessageProvider/status/index.js

```diff
@@ -0,0 +1,75 @@
+const dotenv = require('dotenv').config({ path: `${__dirname}/.env` });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['dotenv' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4499680)